documentation/asciidoc/computers/camera/camera_usage.adoc

```diff
@@ -1,13 +1,9 @@
-== Introducing the Raspberry Pi cameras
+This documentation describes how to program and use supported camera modules with our software tools. All Raspberry Pi cameras can record high-resolution photographs and full HD 1080p video (or better) with our software tools.
```

Comment:
I'm not sure if mentioning "to program" is appropriate here? :shrug:

Reply:
```suggestion
This documentation describes how use supported camera modules with our software tools. All Raspberry Pi cameras can record high-resolution photographs and full HD 1080p video (or better) with our software tools.
```

Excellent point.